generator/src/main/java/com/faforever/neroxis/generator/MapGenerator.java

```diff
@@ -413,7 +419,7 @@ private void encodeMapName() {
                 optionArray = new byte[]{(byte) generatorParameters.spawnCount(),
                                          (byte) (generatorParameters.mapSize() / 64),
                                          (byte) generatorParameters.numTeams(),
-                                         (byte) Biomes.BIOMES_LIST.indexOf(generatorParameters.biome().name()),
+                                         (byte) Biome.getIndexByValue(generatorParameters.biome().name()),
```

Comment:
biome.ordinal()

Reply:
how? The generatorParameters has the actual biome object. The one that stores all the biome settings. And it needs to stay that way because the generatorParameters get referenced in a lot of places to retrieve the actual biome settings.
To get the ordinal we need the enum.

Reply:
If we store the enum in the parameterOptions then we could retrieve it from there, would that work?

Reply:
Yes sorry I missed it wasn't the enum. It should just be the enum

Reply:
Should I maybe rename one of the biome classes to make it less confusing?

Reply:
You don't need a helper method. You can set the biome enum as the input to the option set method like we do with map styles

Reply:
As for the second point, yes you have to use the generatorParameters so we can either put the biome enum  into the generator parameters and delay loading or add the enum to the Biome object class and retrieve it from there.

Reply:
Is the resource path not case sensitive? The convention for enum is to be all upper case, but the folders are not.

Reply:
ping

Reply:
We can set them upper base or just convert the enum

Reply:
how would converting the enum work?

Reply:
By just using a capitalize method that takes an enum as a parameter or some enum to folder name method